packages/js/examples/vanilla/index.html

```diff
@@ -1,88 +1,92 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Signalwire Vanilla Demo</title>
+	<title>Signalwire Call Demo</title>
+	<meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+	<!-- Include the SignalWire Relay JS SDK -->
 	<script type="text/javascript" src="index.min.js"></script>
```

Comment:
is it possible to name it more meaningful like signalwire-relay.min.js ?

Reply:
This will be the hosted file on the CDN... something like:

`cdn.signalwire.com/libs/js/1.1.0/signalwire.min.js`

Reply:
yes, let webpack generate the same name internally same as hosted name so users using the example will have less questions I guess.

Reply:
Do we want the users deal with webpack and the dev setup? I think the best would be replace `index.min.js` with the latest version on our CDN. For example: `cdn.signalwire.com/libs/js/latest/signalwire.min.js` or `../signalwire-latest.min.js`

Reply:
That what I meant, the examples will be copy and paste, not requiring any code to be compiled by the end user... we'll link to the CDN hosted resources.